### ISSUE TITLE
Wrong block count on join

### ISSUE THREAD

**Issue by Nick12506** [TheZoc/sourceforts-bug-tracker#58]
I joined Jake^ on B16 on Melectrome server via the right click on Jakes^ name and joining via the friend method.

I get on, saw we had 8/50. I went to spawn another block and I couldn't. 

I throw one already spawned into a destroyer and then it went to the correct 49/50.

Rejoining showed the correct 49/50.

Not critical IMO though documenting it here.

**Comment by TheZoc** [comment]
I'd need to know a way to reproduce this, to fix it :(

**Comment by Melectrome** [comment]
I was able to reproduce this. Here are the instructions:

1. Max out the block limit to be 50/50, then disconnect and quit the game.
2. Start sourceforts again and join the same server.

You'll notice that the blocklimit says 0/0.
If you just disconnect, do something locally and reconnect to that same server, the bug will occur as well, but there won't be 0/0 but X/50, where X is the blocks you had in the last session.

EDIT: I've just noticed, if you max out the blocklimit and then join a local game, it still says 50/50, even though there are no blocks.
Something gets glitchy when you max out the blocks during a game.
